stack.yaml

```diff
@@ -17,4 +18,5 @@ packages:
 # - doc/tutorial/
 
 extra-deps:
-- hspec-wai-0.10.1
+- hspec-wai-0.11.1
+- pipes-safe-2.3.4
```

Comment:
Possibly unnecessary soon:

* commercialhaskell/stackage/pull/6943

Reply:
Any chance you could hit the CI workflow button?

Reply:
I can't, I don't maintain Servant

Reply:
@tchoutri Would you be able to approve the workflow?